config/settings.py

```diff
@@ -300,25 +300,14 @@
 else:
     HYPERKITTY_DATABASE_URL = ""
 
-# Mailman API credentials
-MAILMAN_REST_API_URL = env("MAILMAN_REST_API_URL", default="http://localhost:8001")
-MAILMAN_REST_API_USER = env("MAILMAN_REST_API_USER", default="restadmin")
-MAILMAN_REST_API_PASS = env("MAILMAN_REST_API_PASS", default="restpass")
-MAILMAN_ARCHIVER_KEY = env("MAILMAN_ARCHIVER_KEY", default="password")
-MAILMAN_ELASTIC_INDEX = env("MAILMAN_ELASTIC_INDEX", default="haystack")
-MAILMAN_HAYSTACK_URL = env("MAILMAN_HAYSTACK_URL", default="http://127.0.0.1:9200/")
```

Comment:
I don't actually see `MAILMAN_CORE_DATABASE` added here. Should it be, or does it only need to be added to the kube files?

Reply:
Right! Pushed another commit.